core/domain/activity_jobs_one_off.py

```diff
@@ -42,31 +42,44 @@ def entity_classes_to_map_over(cls):
     def map(model):
         ids_set = set(model.contributor_ids)
         summary_set = set(model.contributors_summary)
+        modified = False
         if len(ids_set) != len(model.contributor_ids):
             # When the contributor_ids contain duplicate ids.
+            modified = True
+            model.contributor_ids = list(ids_set)
             yield (
                 'DUPLICATE_IDS',
                 (model.id, model.contributor_ids, model.contributors_summary)
             )
         if ids_set - summary_set:
             # When the contributor_ids contain id that is not in
             # contributors_summary.
+            modified = True
+            for user_id in ids_set:
+                model.contributors_summary[user_id] = (
+                    model.contributors_summary.get(user_id, 1))
```

Comment:
How do you know that this value should be 1?

(What does the value actually represent?)

Reply:
If you're asking about the extra parameter in `get(..)`, it's the value that is returned when a KeyError occurs.

Reply:
Ah yep I knew that -- thanks! I actually meant to ask: what is the justification for 1 the correct default value, and could this result in mislabelling?

Reply:
Done, replaced by computing the value.